test/blackbox-tests/test-cases/dune

```diff
@@ -28,6 +28,11 @@
   (deps %{bin:node})
   (alias runtest-js)))
 
+(subdir
+ melange
+ (cram
+  (alias runtest-melange)))
```

Comment:
the tests that execute JS should also depend on node.

Reply:
fixed in https://github.com/jchavarri/dune/pull/5/commits/0b9b266fd8bb8815936764524976b5e6fee86330.